config/BGQ_Clang_ToolChain.cmake

```diff
@@ -1,6 +1,9 @@
 set(CMAKE_C_COMPILER mpiclang) 
 set(CMAKE_CXX_COMPILER mpiclang++11)
 
+# set the linker if extra wrapper is needed for profiling tools like hpctoolkit.
```

Comment:
If we're going to support hpc toolkit it shouldn't be through comments in certain config's. If its just a one time thing put in the manual or wiki and pass it as a -DCMAKE_CXX_LINKER=...

Reply:
Unnecessary. Removed.